scripts/azureml-assets/azureml/assets/get_tokens.py

```diff
@@ -63,7 +66,7 @@ def add_token_info(path: AzureBlobstoreAssetPath, json_info: defaultdict(dict)):
     if container_name in json_info[account_name]:
         return
 
-    _ = path.get_uri(token_expiration=timedelta(days=1))
+    _ = path.get_uri(token_expiration=DEFAULT_SAS_EXPIRATION_TIMEOUT)
```

Comment:
Would you please add a command line argument provide the expiration in days, instead of hard coding it in this script?

Reply:
updated